packages/playground/src/components/pull-request/utils.tsx

```diff
@@ -53,3 +53,38 @@ export function useDateFilters() {
     []
   )
 }
+
+export const timeDistance = (date1 = 0, date2 = 0, onlyHighestDenomination = false) => {
```

Comment:
Can you add an example what this utility is actually doing?

Reply:
added